cognite/powerops/utils/serialization.py

```diff
@@ -1,88 +1,12 @@
 from __future__ import annotations
 
-import contextlib
-import importlib
-import json
-import os
 import re
 import string
 import warnings
-from collections.abc import Iterator
 from pathlib import Path
-from typing import Annotated, Any, ForwardRef, Literal, Union, get_args, get_origin, overload
-
-import tomli_w
-from cognite.client.data_classes import TimeSeries
-from cognite.client.utils._text import to_camel_case
-from pydantic.v1.typing import evaluate_forwardref
-from yaml import CSafeLoader, safe_dump
-
-try:
-    import tomllib  # type: ignore[import-not-found, unused-ignore]
-except ModuleNotFoundError:  # py < 3.11
-    import tomli as tomllib  # type: ignore[import-not-found, unused-ignore]
-
-
-def read_toml_file(toml_file: Path | str) -> dict[str, Any]:
-    """
-    Read a toml file and return a dictionary.
-
-    Args:
-        toml_file: The path to the toml file.
-
-    Returns:
-        A dictionary with the toml data.
-    """
-    return tomllib.loads(Path(toml_file).read_text())
-
-
-def dump_toml_file(toml_file: Path | str, data: dict[str, Any]) -> None:
-    """
-    Dump a dictionary to a toml file.
-
-    Args:
-        toml_file: The path to the toml file.
-        data: The data to dump.
-
-    """
-    Path(toml_file).write_text(tomli_w.dumps(data))
-
-
-@contextlib.contextmanager
-def chdir(new_dir: Path) -> Iterator[None]:

```

Comment:
Oops, I thought I had checked each of them. I'll double-check where/how it's used, might make sense to just put it in functions common because it's a weird thing to have in the SDK

Reply:
Caveat: I just did a quick search in a local verdion of power-ops-functions, so may have found it in a local version of a file, otherwise it sounds good to move it out of the SDK